composer.json

```diff
@@ -34,7 +34,7 @@
         "ext-xml": "*",
         "doctrine/event-manager": "^1.1",
         "jawira/plantuml": "^1.27",
-        "jean85/pretty-package-versions": "^1.2",
+        "jean85/pretty-package-versions": "^1.5 || ^2.0.1",
```

Comment:
Is there any specific reason why we would need to support both versions? If possible we prefer to have just one version :-D

Reply:
Apart from the common problem of installing multiple projects with conflicting constraints, 2.0 requires Composer 2, that's the only difference. If you're fine with imposing that to your end users, you can drop 2.0.

Reply:
Thanks, I will take a look :+1: